msdm/core/problemclasses/mdp/deterministic_shortest_path.py

```diff
@@ -11,14 +11,14 @@ class DeterministicShortestPathProblem(MarkovDecisionProcess):
     """
 
     def next_state_dist(self, s, a) -> Distribution:
-        return Multinomial([self.next_state(s, a)])
+        return DictDistribution({self.next_state(s, a): 1})
```

Comment:
use sites like this make me wonder if we should write a classmethod for constructing uniform distributions, like `DictDistribution.uniform([a, b, c])`

Reply:
yea this would be convenient

Reply:
we should also add `DictDistribution.deterministic('a')` -> `{'a': 1}`